include/bluetooth/services/hids.h

```diff
@@ -125,11 +125,13 @@ typedef void (*bt_gatt_hids_notif_handler_t) (enum bt_gatt_hids_notif_evt evt);
 
 /** @brief HID Report event handler.
  *
- * @param rep  Pointer to the report descriptor.
- * @param conn Pointer to Connection Object.
+ * @param rep	Pointer to the report descriptor.
+ * @param conn	Pointer to Connection Object.
```

Comment:
*connection object?

Reply:
retained the convention in HIDS docs (Control Point, Connection Object etc)

Reply:
ok